internal/v1/server.go

```diff
@@ -469,12 +476,18 @@ func (h *Handlers) ComposeImage(ctx echo.Context) error {
 		return err
 	}
 
-	rawCR, err := json.Marshal(composeRequest)
+	dbComposeRequest := DBComposeRequest{
```

Comment:
Why the separate object?  The idea is to just throw the request in there as it comes in.

Reply:
I made it a separate object to not store the image name in duplicate. I am happy to pass it all in inside of the request but then I don't think we need to store it in a separate column in the database.

Reply:
The reason to store the request in the db is just so users can replay/recreate it easily later. Attributes that we want to maybe index on, and in future potentially search on, and are defined in the api spec (request is just of type {}), we should store those in a proper column imo.

Reply:
Makes a lot of sense. Fixed.